float64 oyaw
float64 rudderP
float64 rudderI
float64 rudderD
float64 sailP
float64 sailI
float64 sailD